tests/__init__.py

```diff
@@ -13,6 +13,9 @@
 os.environ['REDASH_GOOGLE_CLIENT_SECRET'] = "dummy"
 os.environ['REDASH_MULTI_ORG'] = "true"
 
+# Disable login rate limit skip
+os.environ['REDASH_THROTTLE_LOGIN_ENABLED'] = "true"
```

Comment:
If the purpose of the current test is to:
a) test the feature, then I think this should be set in/before the test.
b) make sure prod has this feature turned on, then this should not be set at all.